src/components/modal/Modal.scss

```diff
@@ -150,77 +151,69 @@
     display: flex;
 }
 
-//Responsive support - triggers fullscreen mode after reaching $small-medium-screen width
-.is-responsive-web {
```

Comment:
no changes to this beyond removing the outer `is-responsive-web` block. the styles for `modal-dialog-container` are moved to the block above